tests/unit/controllers/authors-test.js

```diff
@@ -0,0 +1,51 @@
+import {module, test} from 'qunit';
+import {setupTest} from 'ember-qunit';
+import EmberObject from '@ember/object';
+import sinon from 'sinon';
+
+const {spy} = sinon;
+
+module('Unit | Controller | authors', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let controller = this.owner.lookup('controller:authors');
+    assert.ok(controller);
+  });
```

Comment:
That test is not necessary, since none of the other tests will pass unless the controller exists 😄 .

Reply:
Agree. Remove. Thanks.